modules/otel-config-send-metrics-monitoring-stack.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+// 
+// * otel/deploying-otel.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="configuration-for-sending-metrics-to-the-monitoring-stack_{context}"]
+= Configuration for sending metrics to the monitoring stack
+
+The OpenTelemetry Collector custom resource (CR) can be configured to create a Prometheus `ServiceMonitor` CR for scraping the Collector's pipeline metrics and the enabled Prometheus exporters.
+
+.Example of the OpenTelemetry Collector custom resource with the Prometheus exporter
```

Comment:
since you defined CR above, you can use just "CR" down in these headings if you prefer

Reply:
Thank you. I'll think about this change after this PR.